src/index.js

```diff
@@ -24,13 +24,16 @@ function traverseRenderedChildren(internalInstance, callback, argument) {
   }
 }
 
-function setPendingForceUpdate(internalInstance) {
+function setPendingForceUpdate(internalInstance, shouldUpdate) {
+  if (shouldUpdate && !shouldUpdate(internalInstance)) {
```

Comment:
Can you remove the condition and instead pass a `() => true` function by default?